android/app/build.gradle

```diff
@@ -154,14 +154,6 @@ android {
             keyAlias 'androiddebugkey'
             keyPassword 'android'
         }
-        release {
-            if (project.hasProperty('MYAPP_UPLOAD_STORE_FILE')) {
```

Comment:
As you removed this config, we should edit the build process documentation, as we may still want to manually build a signed APK locally in some scenario

Reply:
True. I will check documentation because I'm unsure how to do it at the moment, then add it to the readme once I managed to do it locally

Reply:
Actually, I would like to update the whole readme file, I think some other info has to be updated. Could that be done in another pull request?